infra/bridge/src/utils/webhook-helpers.js

```diff
@@ -0,0 +1,228 @@
+'use strict'
+
+const { decodeHTML } = require('./index')
+
+const logger = require('../logger')
+const crypto = require('crypto')
+
+const request = require('superagent')
+
+let validContents = []
+
+/**
+ * Populates the `validContents` array with contents that can be rewarded
+ */
+module.exports.populateValidContents = async () => {
+  if (!process.env.GROWTH_SERVER_URL) {
```

Comment:
Let's remember to set GROWTH_SERVER_URL in EnvKey before we launch this feature :)

Reply:
Have added it now. Should be good to go after we merge it